src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -91,8 +91,9 @@ public void SslStream_StreamToStream_Successive_ClientWrite_Sync_Success()
         }
 
         [Fact]
-        [ActiveIssue(5896, PlatformID.OSX)]
-        public void SslStream_StreamToStream_LargeWrites_Sync_Success()
+        [InlineData(false)]
+        [InlineData(true)]
```

Comment:
Oops! No, was playing it with and forgot to save the file before submitting... fixed